docs/installation/install-binary.md

```diff
@@ -1011,10 +1011,10 @@ roleRef:
   name: cluster-proxy-clusterrole
 subjects:
   - kind: User
-    name: admin  #这里为证书的用户名，此安装默认为此用户
+    name: admin  # The name should be the user of client certificate, the default value is admin
```

Comment:
The description of the default value is misleading.

Reply:
fixed